plotpuppy/CLI.py

```diff
@@ -281,6 +304,21 @@ def _excepthook(exc_type, value, tb):
     else:
         score = False
 
+    if args.cols:
+        if args.col_order:
+            col_order = args.col_order
```

Comment:
Why do you need a new variable for this?

Reply:
That's a mistake. Fixed